rein/html/plugins/datatables/dataTables.bootstrap.min.js

```diff
@@ -0,0 +1,8 @@
+/*!
```

Comment:
Please commit and link in only the unminified version of this and the datatables css files. Later we can have a build script that minifies but let's keep things unminified for easy/possible review. 

Also, please document the source commit or online generator steps that gave you these files, so independent verification is also possible.